### PR TITLE
New-NsxFirewallRule Added Disabled flag

### PR DESCRIPTION
**Pull request by dcoghlan**
Added the ability to create a disabled distributed firewall rule.
Updated examples to show how to create a disabled rule.
Updated DFW tests to test creation of a disabled rule.

### CODE REVIEW

File: PowerNSX.psm1 [hunk 1]
```diff
@@ -22446,6 +22456,8 @@ function New-NsxFirewallRule  {
         [Parameter (Mandatory=$false)]
             [string]$Comment="",
         [Parameter (Mandatory=$false)]
+            [switch]$DisableRule,
```
Comment (nmbradford) [inline PowerNSX.psm1]: I think I prefer the switch name to be -Disabled rather than -DisableRule.

File: tests/integration/05.Dfw.Tests.ps1 [hunk 2]
```diff
@@ -370,6 +370,23 @@ Describe "Distributed Firewall" {
...
+            @($rule).count | should be 1
+            $rule.sources | should be $null
+            $rule.destinations | should be $null
+            @($rule.appliedToList.appliedto).count | should be 1
+            $rule.appliedToList.appliedTo.Name | should be "DISTRIBUTED_FIREWALL"
+            $rule.appliedToList.appliedTo.Value | should be "DISTRIBUTED_FIREWALL"
+            $rule.appliedToList.appliedTo.Type | should be "DISTRIBUTED_FIREWALL"
+            $rule.name | should be "pester_dfw_rule1"
+            $rule.action | should be allow
+            $rule.disabled | should be $true
```
Comment (nmbradford) [inline tests/integration/05.Dfw.Tests.ps1]: Id prefer a string test rather than boolean ie : should be true, even though im sure this works.

**Comment by dcoghlan** [comment]
Incorporated your requested changes.
All DFW tests now pass.
```
Tests completed in 147.14s
Tests Passed: 67, Failed: 0, Skipped: 0, Pending: 46, Inconclusive: 0```

**Comment by nmbradford** [comment]
Awesome - thanks Dale.

**Pull request merged**